app/Meta.py

```diff
@@ -83,12 +91,13 @@ def insert_experiments(username,fileName):
         logging.error(traceback.format_exc())
         msg = {'Message' : '(psycopg2.errors.UniqueViolation) duplicate key value violates unique constraint "unique_name_per_species"'}
         return make_response(jsonify(msg), 409)
-    except Exception :
+    except Exception as e:
         # Logs the error appropriately
         logging.error(traceback.format_exc())
-        return 410     
+        msg = {'Message' : 'Error' + str(e)}
+        return make_response(jsonify(msg), 410)
      
-def insert_sites(fileName,shp_file,dbf_file,prj_file,shx_file):
+def insert_sites(fileName,shp_file,dbf_file,prj_file,shx_file,status=True):
```

Comment:
Please ensure all the parameters are documented here and elsewhere as well

Reply:
Will make this more clear in the description!